### ISSUE TITLE
Do you also maintain the base / novel splits during pretraining?

### ISSUE THREAD

**Issue by mlzxy** [microsoft/RegionCLIP#63]
Just out of curiosity. Thanks!

**Comment by YiwuZhong** [comment]
During pre-training, there are no so-called "base" or "novel" categories. Our model is pre-trained by a diverse set of object concepts that are parsed from image captions.

**Comment by mlzxy** [comment]
Thx for your clarification. What about the localizer used in pretraining? Is it a rpn trained on base images or a sliding window based proposal generator?

**Comment by YiwuZhong** [comment]
By default, we use RPN trained by the boxes in LVIS dataset (see Implementation details in the paper). Note that, for RPN, there are no base/novel images, since we didn't use categorical labels of these boxes. Also, random boxes can perform closely with RPN (see ablation study in paper).
